search-app/docs/api.md

```diff
@@ -1672,13 +1672,18 @@ Collections can be found by searching for associated services. The following ser
 
 * service_name
   * supports `pattern`, `ignore_case` and option `and`
+* service_type
```

Comment:
Add options description for `service_type`

Reply:
done